app/controllers/admin/base_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::BaseController < ApplicationController
+  before_action :require_admin
+
+  def require_admin
```

Comment:
I think I'm starting to understand...if we want to restrict people who don't have the proper credentials from entering a certain part of our app, that functionality exists in the BaseController.  And is the reason we don't require it for our Merchant because the admin has more privileges then our merchant?

Reply:
I noticed that we do have a merchant base::controller below.  No need to answer this.